databases/sql-server/README.md

```diff
@@ -0,0 +1,191 @@
+
+### 4. Configure the database connection URL
+
+Prisma will use the `DATABASE_URL` environment variable in `.env` to connect to the database.
+Create the file:
+
+```
+touch .env
+```
+
+Then add the following line:
+
+```
+DATABASE_URL=sqlserver://localhost:1433;database=prisma;user=SA;password=prisma123;trustServerCertificate=true;encrypt=true
+```
+
+> **Note:** The database URL uses the `SA` (super admin) user of the database and the same password as defined in `docker-compose.yml`.
+> If you are running on macOS, you may need to use `encrypt=DANGER_PLAINTEXT` to work around the current TLS limitation.
+
+### 5. Introspect your database
+
+The Prisma schema is the foundation for the generated Prisma Client API. Therefore, you first need to make sure that all the database tables are represented in it. The easiest way to do so is by introspecting your database:
+
+```
+npx prisma introspect
+```
+
+> **Note**: You're using [npx](https://github.com/npm/npx) to run Prisma 2 CLI that's listed as a development dependency in [`package.json`](./package.json). Alternatively, you can install the CLI globally using `npm install -g @prisma/cli`. When using Yarn, you can run: `yarn prisma dev`.
+
+The `introspect` command updates your `schema.prisma` file:
+
+```prisma
+model Comment {
+  id        Int      @id @default(autoincrement())
+  createdAt DateTime @default(now())
+  comment   String
+  authorId  Int
+  postId    Int
+  User      User     @relation(fields: [authorId], references: [id])
+  Post      Post     @relation(fields: [postId], references: [id])
+}
+
+model Post {
+  id        Int       @id @default(autoincrement())
+  createdAt DateTime  @default(now())
+  title     String
+  content   String?
+  published Boolean   @default(false)
+  authorId  Int
+  User      User      @relation(fields: [authorId], references: [id])
+  Comment   Comment[]
+  Tag       Tag[]     @relation("TagToPost")
+}
+
+model Tag {
+  id   Int    @id @default(autoincrement())
+  tag  String @unique
+  Post Post[] @relation("TagToPost")
+}
+
+model User {
+  id        Int       @id @default(autoincrement())
+  createdAt DateTime  @default(now())
+  email     String    @unique
+  name      String?
+  Comment   Comment[]
+  Post      Post[]
+}
+```
+
+### 6. Rename the relation fields for easy access
+
+Because both the generated `Post` and `Comment` fields in the `User` model are virtual (i.e. they're not backed by a foreign key in the database), you can manually rename them in your Prisma schema. This will only affect the generated client and is typically done so that they have a more meaningful name in the context of the relation.
+
+Update the schema with the following changes which rename the relation fields:
+
+```prisma
+model Comment {
+  id        Int      @id @default(autoincrement())
+  createdAt DateTime @default(now())
+  comment   String
+  authorId  Int
+  postId    Int
+  author    User     @relation(fields: [authorId], references: [id])
```

Comment:
```suggestion
  writtenBy    User     @relation(fields: [authorId], references: [id])
```

I suggest to change this from `author` to `writtenBy` to avoid duplicate field names (since `Post` already has a field called `author`).